### ISSUE TITLE
Way to manually connect to a device

### ISSUE THREAD

**Issue by Shaniquaniminiquani** [CloakerSmoker/Spotify.ahk#22]
Tl;dr: when spotify opens for the first time it isn't playing anything therefore it is "not connected to a device" therefore I can't change the sound volume etc. A work around I've found to use is to manually send a Media_Next & Media_Prev to catch onto the app player since it starts playback by default when skipping a track and allows spotify.ahk to hook onto it. This allows me to use the commands on spotify's startup like 90% of the time but I've come to realize that if enough time has passed since the last time you opened spotify (I usually always keep it open) it doesn't even have a song to skip ie. 
![image](https://github.com/CloakerSmoker/Spotify.ahk/assets/109847642/abf18fb1-37ba-4be2-a126-7399511e03b1)

So my question basically is, is there a way to manually connect to a device via the api or something? Ot really would only save me like 2 clicks but for an otherwise fully automated experience it's kind of annoying haha. Cheers in advance
